commcare_connect/templates/microplanning/home.html

```
@@ -1,488 +1,508 @@
+                  </dl>
+                </template>
+              </div>
+              <div class="flex gap-2 justify-between">
+                <button class="button button-md outline-style"
+                        @click="showConfirmModal = false"
+                        :disabled="isSaving">{% translate "Back to Edit" %}</button>
+                <button class="button button-md primary-dark"
+                        @click="saveAssignment()"
+                        :disabled="isSaving">
+                  <i x-show="isSaving" class="fa-solid fa-spinner fa-spin mr-1"></i>
+                  <span x-text="isSaving ? '{% translate "Saving..." %}' : '{% translate "Save and Confirm" %}'"></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)